biz_content/forms.py

```diff
@@ -52,3 +52,9 @@ class PermitStatusForm(forms.Form):
     permit_id = forms.CharField(required=True,
                                 label="Permit ID",
                                 help_text="Enter your Permit Application ID")
+
+
+class BizLicenseStatusForm(forms.Form):
+    cu_id = forms.CharField(required=True,
```

Comment:
I added validation for CU, but I wanted to verify all the combinations for a license before adding additional validation.